### ISSUE TITLE
net-analyzer/postgresql_exporter-0.0.2: Bump

### ISSUE THREAD

**Issue by patricklauer** [adjust/gentoo-overlay#173]
Tests fail:
```
>>> Compiling source in /var/tmp/portage/net-analyzer/postgresql_exporter-0.0.2/work/postgresql_exporter-0.0.2 ...
CGO_ENABLED=0 go build -o pg-prometheus-exporter -ldflags "-X=main.version=0.0.2" main.go
>>> Source compiled.
>>> Test phase: net-analyzer/postgresql_exporter-0.0.2
env GOPATH=/var/tmp/portage/net-analyzer/postgresql_exporter-0.0.2/work/postgresql_exporter-0.0.2:/usr/lib/go-gentoo go test -v -work -x github.com/ikitiki/postgresql_exporter
can't load package: package github.com/ikitiki/postgresql_exporter: found packages config (config.go) and db (db.go) in /var/tmp/portage/net-analyzer/postgresql_exporter-0.0.2/work/postgresql_exporter-0.0.2/src/github.com/ikitiki/postgresql_exporter
 * ERROR: net-analyzer/postgresql_exporter-0.0.2::adjust failed (test phase):
 *   (no error message)
```
/cc @ikitiki

**Comment by ikitiki** [comment]
as for now I'd remove test phase

**Comment by patricklauer** [comment]
Fixed in #175
